COMPILING.md

```diff
@@ -64,47 +65,57 @@ make
 sudo make install # optional
 ```
 
-To control the server with systemd, see this [unit file example](https://github.com/jamulussoftware/jamulus/blob/master/distributions/debian/jamulus-headless.service). See also runtime [configuration options](https://jamulus.io/wiki/Running-a-Server#command-line-options), and [this manual](https://jamulus.io/wiki/Server-Linux).
+To control the server with systemd, runtime options and similar, refer to the [Server manual](https://jamulus.io/wiki/Server-Linux).
```

Comment:
Server-Linux.md won't exit after the next website release, but I think this will re-direct OK. Might want to replace it with a slightly nicer anchor link in the revised doc though.

Reply:
So it'll be "running a server "then? Could you please add the correct suggestion + anchor then?

Reply:
Sorry, I meant we should just leave it as a redirect for now until we have the revised server docs. It's all going to be completely re-written: https://github.com/jamulussoftware/jamuluswebsite/issues/729